.github/workflows/verify_chromatic.yml

```diff
@@ -31,7 +31,7 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
         with:
-          fetch-depth: 0 # Required to retrieve git history
+          fetch-depth: 1000 # Required to retrieve git history
```

Comment:
I'd actually up it a bit, maybe 5000 or 10000. Sometimes PRs can come from pretty far behind